AdminOpsGuide/dbrepl.rst

```diff
@@ -942,19 +942,21 @@ Here is the code:
    export ydb_principal_editing=EDITING
    export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist"
    #export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist/libgtmutil.so"
-   # Here is an example of setting the ydb_routines environment variable:
-   # if [ -e  "$ydb_dist/libgtmutil.so" ] ; then export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist/libgtmutil.so"
-   else export ydb_routines="$PWD/$ydb_repl_instname* $ydb_dist" ; fi
-   # For more examples on setting YottaDB related environment variables to reasonable values on POSIX shells, refer to the ydb_set_env script.
    #export LD_LIBRARY_PATH=/usr/local/lib
    #export ydb_crypt_config=$PWD/$ydb_repl_instname/config_file
    #echo -n "Enter Password for ydb_tls_passwd_${ydb_repl_instname}: ";export ydb_tls_passwd_${ydb_repl_instname}="`$ydb_dist/plugin/gtmcrypt/maskpass|tail -n 1|cut -f 3 -d " "`"
 
-Modify the env script according to your test environment. 
+.. note::
+   Here is an example of setting the ydb_routines environment variable:
```

Comment:
What is the purpose of this line? It does not render well either (comes in as a continuous paragraph and is hard to read in a doc).